### ISSUE TITLE
support `AWS_ENDPOINT_URL` env variable

### ISSUE THREAD

**Issue by MrFishFinger** [awslabs/mountpoint-s3#890]
### Tell us more about this new feature.

it would be great if the tool supported the `AWS_ENDPOINT_URL` environment variable, in a similar way to the official `aws-cli` does.

see:
- https://docs.aws.amazon.com/sdkref/latest/guide/settings-reference.html#EVarSettings
- https://github.com/aws/aws-cli/issues/8052

```
# works
❯ mount-s3 --endpoint-url=https://s3.eu-central-003.backblazeb2.com  test-b2-bucket /mnt/s3/test-b2-bucket
bucket test-b2-bucket is mounted at /mnt/s3/test-b2-bucket


# doesnt work :(
❯ export AWS_ENDPOINT_URL=https://s3.eu-central-003.backblazeb2.com
❯ mount-s3 test-b2-bucket /mnt/s3/test-b2-bucket
Error: Failed to create S3 client

Caused by:
    0: initial ListObjectsV2 failed for bucket test-b2-bucket in region us-east-1
    1: Client error
    2: Forbidden: The AWS Access Key Id you provided does not exist in our records.
Error: Failed to create mount process
```

**Comment by arsh** [comment]
This enhancement has been merged https://github.com/awslabs/mountpoint-s3/pull/895. If you build from source, you can use now. Otherwise, it will be available on the next release.

**Comment by dannycjones** [comment]
This change was released in v1.7.0 published today. 🎉 

https://github.com/awslabs/mountpoint-s3/releases/tag/mountpoint-s3-1.7.0
